web/modules/custom/hashtags/README.txt

```diff
@@ -0,0 +1,47 @@
+Hashtags module allows to use the hashtags for the content entities.
```

Comment:
@vishalbandre since we are using https://git.drupalcode.org/project/hashtags/-/tree/8.x-1.x, i would recommend to include the project via composer and add a patch to the project for the changes done by you specifically.

Reply:
Sure Hemant will make the change. Thank you.